app/assets/javascripts/initializers/initializeCommentsPage.js.erb

```diff
@@ -270,6 +270,34 @@ function handleKeyUp(event) {
   handleSizeChange(event);
 }
 
+function handleKeyDown(event) {
+    // If Ctrl+Enter OR Command+Enter is pressed
+    if((event.ctrlKey || event.metaKey) && event.keyCode === 13)
+    {
+        // Get user details and extract code of conduct & comment count
+        var user = userData();
+        if (!user){
+            return;
+        }
+        //var codeOfConduct = user.checked_code_of_conduct;
+        var commentCount = user.number_of_comments;
+
+        // Ignoring --> Verify that code of conduct has has been accepted
+        // since it is false the first time even if the user has accepted it
+
+        // Ensure that user has commented at least once
+        // and we have a non-empty comment to post
+        if (//codeOfConduct &&
+            commentCount >= 1 &&
+            event.target.value.trim() !== '') {
```

Comment:
I think that according to [airbnb style guide](https://github.com/airbnb/javascript#control-statements), we should write this with a syntax similar to:
```javascript
if (
  condition1
  && condition2
  && ...
) {
  // code
}
```
Thoughts?

Reply:
I prefer the other way. That's what prettier recommends by default as well I think. This should be in the .eslint file if it isn't already.